Tests/verify_checkout_functionality.py

```diff
@@ -0,0 +1,174 @@
+import unittest
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service as ChromeService
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+from pages.home_page import HomePage
+from pages.login_page import LoginPage
+from pages.my_account_page import MyAccountPage
+from pages.product_page import ProductPage
+from pages.cart_page import CartPage
+from pages.checkout_page import CheckOutPage
+
+
+class TestEndToEndPurchase(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.browser = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()))
+
+    def test_end_to_end_purchase(self):
+        browser = self.browser
+
+        browser.get("http://demostore.supersqa.com/")
+
+        homepage = HomePage(browser)
```

Comment:
this is great having all of them together so you can know in one spot what you are calling into the tests